dev-python/toposort/toposort-1.6.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
```

Comment:
Can we bump this to `EAPI=8`? And does this package work with `python3_10`?

Reply:
Yes, it looks working for python-3.10:
```
 * python3_10: running distutils-r1_run_phase python_test
python3.10 setup.py test --verbose
running test
..........
----------------------------------------------------------------------
Ran 10 tests in 0.002s

OK
test suite
doctests
test complete
 * python3_10: running distutils-r1_run_phase _distutils-r1_clean_egg_info
>>> Completed testing dev-python/toposort-1.6
```